### PR TITLE
RECIPE-125

### PR DESCRIPTION
**Pull request by PatsiChang**
Fix unit test by creating new service for getting ENV values 
Link: https://smart-home-cloud.atlassian.net/browse/RECIPE-125

### CODE REVIEW

File: src/main/java/com/patsi/service/RecipeEnvValueService.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public Supplier<Boolean> getRecipeFileFlag() {
+        return () -> transferRecipeFileFlag;
+    }
+
+    public Supplier<String> getImgPath() {
+        return () -> IMAGE_PATH;
+    }
+
+    public Supplier<String> getImgStagedPath() {
+        return () -> IMAGE_STAGED_PATH;
```
Comment (abcboyabcd) [inline src/main/java/com/patsi/service/RecipeEnvValueService.java]: why using supplier?

File: src/main/resources/application-test.properties [hunk 2]
```diff
@@ -1,6 +1 @@
-spring.datasource.url=jdbc:h2:mem:testdb
```
Comment (abcboyabcd) [inline src/main/resources/application-test.properties:1]: why deleting these test properties ?

**Review by abcboyabcd — APPROVED** [review]

**Review by ChakShingLo — APPROVED** [review]

File: src/main/resources/application-test.properties (hunk 2 above)
Comment (PatsiChang) [reply src/main/resources/application-test.properties:1]: changing to postgres:)

File: src/main/java/com/patsi/service/RecipeEnvValueService.java (hunk 1 above)
Comment (PatsiChang) [reply src/main/java/com/patsi/service/RecipeEnvValueService.java]: Fixed :D

**Pull request merged**